core/src/main/java/io/grpc/internal/AbstractManagedChannelImplBuilder.java

```diff
@@ -192,8 +202,7 @@ public ManagedChannelImpl build() {
         buildTransportFactory(), authorityOverride);
     return new ManagedChannelImpl(
         target,
-        // TODO(carl-mastrangelo): Allow clients to pass this in
-        new ExponentialBackoffPolicy.Provider(),
+        firstNonNull(backoffPolicyProvider, new ExponentialBackoffPolicy.ReconnectProvider()),
```

Comment:
You were planning to use this for ClientCalls correct?  Currently this is used for connection backoff.  I can't think of any immediate reason why someone would want separate backoffs, but someone might.  (Connections are retried indefinitely, but calls fail after 3 tries, or something like that).